package.json

```diff
@@ -18,14 +22,15 @@
     "build:docker": "docker-compose -f docker-compose.yml up -d --build",
     "test": "BRANCH=local AWS_XRAY_CONTEXT_MISSING=LOG_ERROR jest --testMatch=\"**/*.unitTest.ts\" --coverage",
     "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
+    "test-local-i": "USE_CONTAINERIZED_DATABASE=1 BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
     "tslint": "tslint src/**/*.ts tests/**/*.ts -q",
     "prepush": "npm test && npm run tslint",
```

Comment:
I think we can change the hooks order like we have on update store however a few things are missing i think:
- commitlint
- prettier
- eslint
- scripts around the automation of the release(s)

https://github.com/dvsa/cvs-svc-enquiry/blob/feature/CVSB-19222/package.json#L11

Reply:
- `commitlint` will not allow a single one of the commit messages I have made on this repo, as well as involving an extra set-up step to make sure environment variable `HUSKY_GIT_PARAMS` is set. Perhaps consider adding this at a later date? I want to make this release cut happen :)
- `prettier`: yes, I can add this. exactly 60 files will change as a result of this commit. Let me know if you want this to happen.
- `eslint` is superseded by `tslint`, as far as I know, so i'm not adding `eslint` on top
- scripts: isn't this done already? the pipeline works fine with the existing tools-setup
- envc configuration: same story as the above - please let me know if this is not the case

Reply:
thanks for clarifying, I appreciate and feel free to resolve guys if you think it is ok.